test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -154,6 +154,8 @@ public void TensorFlowTransforCifarEndToEndTest2()
             Assert.Equal(0, prediction.PredictedScores[0], 2);
             Assert.Equal(0, prediction.PredictedScores[1], 2);
             Assert.Equal(1, prediction.PredictedScores[2], 2);
+
+            transformer.Dispose();
```

Comment:
You can use the `using` statement in all these tests. It will make this cleaner.

[https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement)